tests/stress_tests/memcheck_tests/tests.cpp

```diff
@@ -106,3 +106,74 @@ INSTANTIATE_TEST_CASE_P(MemCheckTests, MemCheckTestSuite,
                         ::testing::ValuesIn(
                                 generateTestsParams({"devices", "models"})),
                         getTestCaseName);
+
+TEST_P(MemCheckTestSuite, inference_with_streams) {
+    const auto nstreams = "2";
```

Comment:
Why number of streams 2  is selected?

Reply:
At the moment we don't want to check different values of streams, so `2` is some simple case. If required, we may increase it to have more visible changes in memory consumption